src/PrivateDns/PrivateDns/Models/PSPrivateDnsRecordSet.cs

```diff
@@ -0,0 +1,583 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
Please get a team member of yours to sign off on the next two files - they look okay to me, but I do not completely understand the functionality so I might be missing something.

Reply:
Reviewed and posted comments. I think we can close this comment now.